speech-to-text.js

```diff
@@ -17,14 +17,25 @@
   const JSONStream = require("JSONStream");
 
   const filename = process.argv[2];
+  let language = '';
+
   if (!filename) {
     exitLog(`
       usage: node speech-to-text.js <path_to_wav_file> [<language>]
 
       <path_to_wav_file>: required. it's the path to the wav file to be transcribed.
       <language>: optional. e.g. en-US
+        Language may be inferred from filename if it contains text mathcing locale-(locale).
```

Comment:
@rtrvrtg small typo for "mathcing" on this line.

Otherwise, I think all looks good.